presto-hive/src/main/java/io/prestosql/plugin/hive/HiveConfig.java

```diff
@@ -79,6 +80,7 @@
     private HiveCompressionCodec hiveCompressionCodec = HiveCompressionCodec.GZIP;
     private boolean respectTableFormat = true;
     private boolean immutablePartitions;
+    private String insertExistingPartitionsBehavior;
```

Comment:
Please remove `immutablePartitions`, notice that it has to be then added to `@DefunctConfig` at the top of the class, so it raise proper error message when some uses it.

`immutablePartitions` usage should be now replaced with the new property.

Please update `TestHiveConfig`

Reply:
immutablePartitions=true is used to prohibit to set insertExistingPartitionsBehavior to OVERWRITE via session properties. insertExistingPartitionsBehavior is not a superset of immutablePartitions. So I will left it.

TestHiveConfig is updated.